app/src/main/java/com/nervousfish/nervousfish/side_activities/LoginActivity.java

```diff
@@ -1,11 +1,13 @@
-package com.nervousfish.nervousfish;
+package com.nervousfish.nervousfish.side_activities;
```

Comment:
Because we only want a single Service Locator, made by the very first Activity created. To make sure that only that very first activity, namely MainActivity, can create a new one, we put the MainActivity in the same package as the ServiceLocator classes and make the constructor package-private. The other activities cannot create a second service locator anymore when they are in a seperate package and should get the service locator dependency injected by MainActivity